typings/externs.d.ts

```diff
@@ -219,6 +220,7 @@ declare global {
           styleSheetUrl?: string;
           timerId?: string;
           url?: string;
+          processId?: number;
```

Comment:
why did this need to be added aren't we still accessing the same `processId` in `frames` array?

Reply:
It isn't needed for this. I forgot to remove this (from the data uri changes).

`findTracingIds` doesn't work when you use a data uri, since the PID changes, and thus there is no metadata for the expected pid for the thread `CrRendererMain`. But, there is an event we can use to match the frameId (which doesn't change!) to the new pid (see https://github.com/GoogleChrome/lighthouse/blob/undo-revert-datauri/lighthouse-core/lib/traces/tracing-processor.js#L225 ). TL;DR - `processId` is just on the event `FrameCommittedInBrowser`, which is not needed for `about:blank`.